install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml

```diff
@@ -164,6 +164,11 @@ data:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+{{- if .Values.global.podDNSSearchNamespaces }}
+        - name: ISTIO_META_DNS_DOMAINS
+{{- $local := dict "first" true }}
+          value: {{ range $k, $v := .Values.global.podDNSSearchNamespaces }}{{- if not $local.first }}, {{ end }}{{- $v }}{{- $_ := set $local "first" false }}{{- end }}
+{{- end }}
```

Comment:
You should be adding the default dns domain as well (the svc.cluster.local or whatever. I think there is a global value for it)

Reply:
This value it is part of Values.global.podDNSSearchNamespaces, although at this time the patch fails with it present - so I removed it temporarily.  The exact text is `  - "[[ valueOrDefault .DeploymentMeta.Namespace \"default\" ]].global" `.  I intend to revert its removal (IOW restore it) before final merging.

Reply:
Did you mean `$(POD_NAMESPACE).svc.cluster.local`?  If so, that may solve a different problem I was suffering with this PR.  Will try it out!

Reply:
I added this ${POD_NAMESPACE}.svc.cluster.local here, and the result was a duplicated entry received by the application proxy.  Any idea why?